MCLOUD-6708: Add notice that only Production mode is supported on cloud (#7728)
magento/devdocs: src/_includes/cloud/error-codes.md

| 2023 | install-update:split-db | Enabling a split database will be skipped. |  |
| 2024 | install-update:split-db | The SPLIT_DB variable is missing the configuration for split connection types. |  |
| 2025 | install-update:split-db | Slave connection not set. |  |
| 2026 | pre-deploy:restore-writable-dirs | Failed to restore some data generated during the build phase to the mounted directories | Check the `cloud.log` for more information. |

> | 2027 | validate-config:mage-mode-variable | Environment variable MAGE_MODE has value with not supported mode  | Remove MAGE_MODE environment variable or change its value to "production". Magento Cloud supports only "production" mode. |

Reviewer: Suggest changing the error code description for 2027 in the code to:

The mode value specified in the MAGE_MODE environment variable is not supported.

Author: Updated

> | 2027 | validate-config:mage-mode-variable | The mode value specified in the MAGE_MODE environment variable is not supported. | Remove MAGE_MODE environment variable or change its value to "production". Magento Cloud supports "production" mode only. |

Reviewer: ```suggestion
| 2027 | validate-config:mage-mode-variable | Mode value for MAGE_MODE environment variable not supported | Remove the MAGE_MODE environment variable, or change its value to "production". Magento Cloud supports "production" mode only. |
```
